src/main/java/core/basesyntax/strategy/ActivityStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activity.ActivityHandler;
+import java.util.Map;
+
+public class ActivityStrategyImpl implements ActivityStrategy {
+
```

Comment:
```suggestion
```

Reply:
> 

![Screenshot_8](https://github.com/mate-academy/jv-fruit-shop/assets/54332495/96c99fce-1a00-403f-bb20-5815d9f871a2)
)

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/156338484/c43a3bf9-1cc6-4db4-9023-a4bf67cf3c04)
![image](https://github.com/mate-academy/jv-fruit-shop/assets/156338484/d6bac84e-5474-4451-b032-e03c76c19bd9)

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/156338484/d69b5526-816f-4320-a4d8-98e74980cbf6)
end it`s not start from new line